documents/src/pages/utils/date-time.md

```diff
@@ -5,8 +5,1270 @@ type: page
 layout: default
 -->
 
-::status-todo::
+::status-complete::
 
 # Date & Time
 
-::todo::
+The utility provides the comprehensive set of tools to manipulate with date/time/datetime strings. It overcomes the problems with JavaScript `Date` implementation, such as:
+
+- string to date parsing is error-prone and can be different in browsers
+- Date object is mutable
+- daylight saving behavior is unpredictable
+- computation APIs are unwieldy or unavailable
+- defining and parsing dates in _UTC_, _Local_ or _Developer Defined_ timezones is error-prone
```

Comment:
```suggestion
- Defining and parsing dates in _UTC_, _Local_ or _Developer Defined_ timezones is error-prone
```

Reply:
I'm seeing a lot of your descriptions not starting capitalised. Is this on purpose? If so, how come?

Reply:
previous sentence ends with semi column (`...implementation, such as:`), it means that the list should not be capitalised. `Date` is capitalised because it is the name of the object

Reply:
Ok what about descriptions in tables?

Reply:
Same, if the name, than capitalised, e.g. `Number` or `String`, otherwise lower case

Reply:
It looks odd for some reason. Just looked at what Lit are doing and they Capitalise each line.

Reply:
![image](https://user-images.githubusercontent.com/1519516/179698886-10fe47dc-9579-4710-b873-b087198055e6.png)

Reply:
date-fns does not

Reply:
I think it is the matter of consistency really, and for consistency with other pages I will capitalise them :)